packages/block-library/src/navigation-link/link-ui.js

```diff
@@ -176,11 +98,11 @@ function UnforwardedLinkUI( props, ref ) {
 
 	const dialogTitleId = useInstanceId(
 		LinkUI,
-		`link-ui-link-control__title`
+		'link-ui-link-control__title'
 	);
 	const dialogDescriptionId = useInstanceId(
 		LinkUI,
-		`link-ui-link-control__description`
+		'link-ui-link-control__description'
```

Comment:
What is the difference between backticks and single quotes in this context?

Reply:
Just an error when I remove some interpolation from this string. Single quotes are preferred. Suprised the linter doesn't pick up on it!